src/lib/components/colors/index.svelte

```diff
@@ -10,13 +10,13 @@
 
 <SupportWarning format={$format} />
 
-<form data-form="contrast-checker" data-layout="color-form">
+<div data-form="contrast-checker" data-layout="color-form">
```

Comment:
This was nesting a form inside a form (Sliders are forms), and Svelte wasn't even loading the page. This moves the Header input outside of a form- we'll need to figure out the rest place for forms.

Reply:
Updated in https://github.com/oddbird/oddcontrast/commit/a79e99a23789e50bc5b565788cbcc5a20fecabf5